### PR TITLE
Feature/team view

### PR DESCRIPTION
**Pull request by YaelBroudy**

### CODE REVIEW

File: src/app/team-view/page.js [hunk 1]
```diff
@@ -321,19 +321,34 @@ console.log(data)
               <td style={{backgroundColor: Colors[3][2]}}>Side</td>
             </tr>
             <tr>
-              <td style={{backgroundColor: Colors[3][3]}}>{Math.round(1000*data.endgame.onstagePlacement.center)/10+"%"}</td>
-              <td style={{backgroundColor: Colors[3][3]}}>{Math.round(1000*data.endgame.onstagePlacement.side)/10+"%"}</td>
+              <td style={{backgroundColor: Colors[3][3]}}>{Math.round(10*data.endgame.onstagePlacement.center)/10+"%"}</td>
+              <td style={{backgroundColor: Colors[3][3]}}>{Math.round(10*data.endgame.onstagePlacement.side)/10+"%"}</td>
```
Comment (pseay) [inline src/app/team-view/page.js:325]: looks good! Glad it's rounded right. mb there 😬

File: src/app/team-view/page.js [hunk 2]
```diff
@@ -321,19 +321,34 @@ console.log(data)
...
             </tr>
           </table>
-          <table className={styles.intakeTable}>
+          <table className={styles.differentTable}>
+            <tr>
+              <td style={{backgroundColor: Colors[3][1], width:"30px"}} rowSpan="4">Trap</td>
```
Comment (pseay) [inline src/app/team-view/page.js:330]: its a trap!

**Review by pseay — COMMENTED** [review]
Looking pretty good :)

File: src/app/team-view/page.js (hunk 2 above)
Comment (YaelBroudy) [reply src/app/team-view/page.js:330]: it sure is!!!

**Review by YaelBroudy — COMMENTED** [review]
thanks :)

**Review by pseay — APPROVED** [review]
My name is Preston Seay and I approve of this message!

**Pull request merged**